infra/gcp/terraform/k8s-infra-oci-proxy-prod/terraform.tfvars

```diff
@@ -146,103 +146,103 @@ cloud_run_config = {
       },
       {
         name  = "UPSTREAM_REGISTRY_PATH",
-        value = "k8s-artifacts-prod"
+        value = "k8s-artifacts-prod/images"
```

Comment:
Non-blocking because this looks correct, but out of curiosity why don't we have `https://europe-west3-docker.pkg.dev`?

Reply:
we should move this to the nearest AR region.

I'm not sure how the AR regions were selected, I think it just got out of sync with the cloud run regions.

Reply:
need to determine which is closest out of the ones we have.

Reply:
Frankfurt (europe-west3)

London(europe-west2)
Belgium(europe-west1) 

Presumably europe-west1

Reply:
I've set this to europe-west1 with a TODO

Reply:
It should be 1-to-1 mapping. The GCLB will work out the closest region for us. The region selection was basically all of the Tier 1 regions and a selection of distant Tier 2 regions. I still think we need to add more regions but the image promoter is kinda slow today so we'll need to revisit it once the image promoter is optimised.

I forgot about the AR Registry for Frankfurt. We can disable the cloud run service in that region.

Reply:
Dropping the region sounds reasonable to me

Reply:
done in the next commit